tests/common/nebula_test_suite.py

```diff
@@ -0,0 +1,159 @@
+            if isinstance(expect, Pattern):
+                if not expect.match(str(col.get_integer())):
+                    return False, msg
+            else:
+                if col.get_integer() != expect:
+                    return False, msg
+            return True, ''
+
+        if col.getType() == ttypes.ColumnValue.ID:
+            msg = self.check_format_str.format(col.getid(), expect)
+            if isinstance(expect, Pattern):
+                if not expect.match(str(col.gettid())):
+                    return False, msg
+            else:
+                if col.gettid() != expect:
+                    return False, msg
+            return True, ''
+
+        if col.getType() == ttypes.ColumnValue.STR:
+            msg = self.check_format_str.format(col.get_str().decode('utf-8'),
+                                               expect)
+            if isinstance(expect, Pattern):
+                if not expect.match(col.get_str().decode('utf-8')):
+                    return False, msg
+            else:
+                if col.get_str().decode('utf-8') != expect:
+                    return False, msg
+            return True, ''
+
+        if col.getType() == ttypes.ColumnValue.DOUBLE_PRECISION:
+            msg = self.check_format_str.format(col.get_double_precision(),
+                                               expect)
+            if isinstance(expect, Pattern):
+                if not expect.match(str(col.get_double_precision())):
+                    return False, msg
+            else:
+                if not math.isclose(col.get_double_precision(), expect):
+                    return False, msg
+            return True, ''
+
+        if col.getType() == ttypes.ColumnValue.TIMESTAMP:
+            msg = self.check_format_str.format(col.get_timestamp(), expect)
+            if isinstance(expect, Pattern):
+                if not expect.match(str(col.get_timestamp())):
+                    return False, msg
+            else:
+                if col.get_timestamp() != expect:
+                    return False, msg
+            return True, ''
+        return False, 'ERROR: Type unsupported'
+
+    @classmethod
+    def search_result(self, rows, expect):
+        msg = 'len(rows)[%d] != len(expect)[%d]' % (len(rows), len(expect))
+        assert len(rows) == len(expect), msg
+        for row in rows:
+            ok = False
+            msg = ""
+            for exp in expect:
+                for col, i in zip(row.columns, range(0, len(exp))):
+                    ok, msg = self.check_value(col, exp[i])
+                    if not ok:
+                        break
+                if ok:
+                    break
+            assert ok, msg
+            expect.remove(exp)
+
+    @classmethod
+    def check_result(self, rows, expect):
+        msg = 'len(rows)[%d] != len(expect)[%d]' % (len(rows), len(expect))
+        assert len(rows) == len(expect), msg
+        for row, i in zip(rows, range(0, len(expect))):
```

Comment:
need to support  `result is the same, but the order of the rows is not the same`

Reply:
we using "search_result" check this condition.

Reply:
good